src/htm/types/Types.hpp

```diff
@@ -97,6 +97,11 @@ typedef void* Handle;
  */
 typedef std::size_t Size;
 
+/**
+ * Regpresents a boolean value.
+ */
+typedef bool Bool;
```

Comment:
I needed to be able to create getPrameterBool( ) with a macro.  Otherwise it was getParameterbool() and I had to define a function with that name that called getPrameterBool().